### ISSUE TITLE
Shadow rendering

### ISSUE THREAD

**Issue by stphil1** [CesiumGS/cesium#9924]
Dear all

We observed some misleading visual effects with the shadow rendering using the [virtualcitysystems 3D](https://vc.systems/en/products/vc-planner/) Suite for urban planning and would like to open an issue ticket. Virtualcitysystems informed us, that they are using the Cesium version 79 in the software.
I attached some shadow samples to the ticket. Also I would like to share the link to my initial post on the [community forum](https://community.cesium.com/t/shadow-rendering/15980).

Is there some way to avoid these problems? You can encounter the false rendering manner in every perspective, while zooming and panning. We found an old post (https://community.cesium.com/t/3d-tiles-shadows/4289) aiming at the same issue.

Browser: Firefox 94.0.1 (64-Bit), Chrome 95.0.4638.69 (64-Bit), Edge 95.0.1020.44 (64-Bit)
Operating System: Windows 10, 64-bit, The 3D hardware acceleration on this computer is activated.

If you have any Ideas what we can do to avoid or fix these issues we would appreciate it a lot.
Kind regards and many thanks in advance for taking the time to investigate this issue.

Philippe

![shadow_rendering_issue_1](https://user-images.githubusercontent.com/93985942/140926846-16c5bc5d-a24f-49d5-a033-c188a42c543d.PNG)
![shadow_rendering_issue_2](https://user-images.githubusercontent.com/93985942/140926852-0d44fd5a-cdb6-4550-9bb9-91e9b0310ce7.PNG)
![shadow_rendering_issue_3](https://user-images.githubusercontent.com/93985942/140926856-a05c55d2-3e76-4b90-ab9a-593e7e989eb1.PNG)
![shadow_rendering_issue_4](https://user-images.githubusercontent.com/93985942/140926860-7afb4fb8-3b58-4967-9b7d-5d470a4fe441.PNG)
![shadow_rendering_issue_5](https://user-images.githubusercontent.com/93985942/140926863-913493a2-b7b5-486e-b38d-66b0426ebd21.PNG)

**Comment by srothst1** [comment]
Thank you @stphil1!

**Comment by ebogo1** [comment]
@stphil1 For now there's no definite workaround for shadow map artifacts. But, some of them can be mitigated with the `maximumDistance` and `size` properties (see the [ShadowMap docs](https://cesium.com/learn/cesiumjs/ref-doc/ShadowMap.html)).

**Comment by virtualcitywendt** [comment]
@stphil1 I totally agree. I've got the same feedback regarding the shadow artefacts from hundreds of users. Unfortunately changing the parameters of  `maximumDistance` and `size ` works just for certain situations and isn't that satisfying at all. A more sustainable global solution would be the right one with which we woud be grateful. Please keep us updated regarding the issue :) 

Best,
Tim

**Comment by seraina-silberschmidt** [comment]
Dear all,
Is there an update on this matter? 
As for now the shadow artefacts persist, creating rather unpleasant results:

![image](https://user-images.githubusercontent.com/67998538/151125172-95c4e7e8-4f25-40e0-bb97-a9c53cdf08c4.png)
![image](https://user-images.githubusercontent.com/67998538/151126670-78299b83-f134-4ac3-ba42-4b72dbb9f8c9.png)


The occurrence and severity of these artefacts seem to depend on the selected date and time as well as the camera angle and distance in the 3D-map.

We would be very pleased if you could prioritize this issue.

Many thanks and best regards,
Seraina

**Comment by virtualcitywendt** [comment]
Dear Cesium Team,

can you please give an update on this issue? Currently, we have our user conference in Berlin and are talking with the community about this issue and wondering if there is any progress. 

Thank's a lot in advance,

Best,
Tim
